Source/WebCore/Modules/filesystemaccess/WorkerFileSystemStorageConnection.cpp

```diff
@@ -313,6 +313,68 @@ void WorkerFileSystemStorageConnection::invalidateAccessHandle(WebCore::FileSyst
         handle->invalidate();
 }
 
+void WorkerFileSystemStorageConnection::createWritable(FileSystemHandleIdentifier identifier, bool keepExistingData, VoidCallback&& callback)
```

Comment:
Yes, I thought about that, but since the patch is big, might be easier to keep the new functions consistent with existing functions, and do the refactoring for all functions later.